### PR TITLE
Update foreman nodejs dependencies

### PR DESCRIPTION
**Pull request by ekohl**
This replaces https://github.com/theforeman/foreman-packaging/pull/2001 and is mostly based on a script output. I'll further automate that and add it as a helper.

### CODE REVIEW

File: foreman/foreman.spec [hunk 1]
```diff
@@ -223,6 +223,12 @@ BuildRequires: libuv
 BuildRequires: nodejs-packaging
 
 # package.json devDependencies
+#BuildRequires: npm(@storybook/addon-actions) >= 3.2.12
+#BuildRequires: npm(@storybook/addon-actions) < 4.0.0
+#BuildRequires: npm(@storybook/react) >= 3.2.12
```
Comment (dLobatog) [inline foreman/foreman.spec]: huh?

Comment (ekohl) [reply foreman/foreman.spec]: I'm not sure what to do about these - they've been here for a while but we never packaged storybooks until now.

**Comment by ekohl** [comment]
Going to create PRs for missing nodejs dependencies.

**Comment by ekohl** [comment]
Was missing some releases and https://github.com/theforeman/foreman-packaging/pull/2041

**Comment by ekohl** [comment]
[test]

**Comment by ekohl** [comment]
https://github.com/theforeman/foreman-packaging/pull/2042 we could use repoclosure tests

**Comment by ekohl** [comment]
[test]

**Pull request merged**